src/pages/en/concepts/mpa-vs-spa.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: MPAs vs. SPAs
+description: TODO
+i18nReady: true
+---
+
+Understanding the tradeoffs between Multi-Page Application (MPA) and Single-Page Application (SPA) architecture is key to understanding what makes Astro different from other web frameworks like Next.js and Remix.
+
+## Terminology
+
+**A Multi-Page Application (MPA)** is a website consisting of multiple HTML pages, mostly rendered on a server. When you navigate to a new page, your browser requests a new page of HTML from the server. **Astro is an MPA framework.** More traditional MPA frameworks also include Ruby on Rails, Python Django, PHP Laravel, Wordpress, and static site builders like Eleventy, Hugo and VitePress.
+
+**A Single-Page Application (SPA)** is a website consisting of a JavaScript application that loads in the user's browser and then renders HTML locally. SPAs may *also* generate HTML on the server (especially for the the first page load) but SPAs are unique in their ability to render HTML locally when you navigate to a new page. Next.js, Nuxt, SvelteKit, Remix, Gatsby, and Create React App are all examples of SPA frameworks.
+
+## Astro vs. other MPAs
+
+Astro is an MPA framework. However, Astro is unique from other MPA frameworks due to its use of JavaScript as a server runtime. Traditional MPA frameworks would have you write one language on the server (Ruby, PHP, etc.) and JavaScript on the browser. In Astro, you're always writing JavaScript. You can also re-use UI components (like React and Svelte) on both the server and the client. 
+
+The result is that Astro has a developer experience similar to that of Next.js or another modern SPA, even if the user experience is an entirely different MPA site architecture.
+
+## MPAs vs. SPAs
+
+There are three main differences to be aware of when comparing MPAs vs. SPAs:
+
+#### Server rendering (MPA) vs. client rendering (SPA)
+
+In MPAs, most of your HTML is rendered on the server. In SPAs, most HTML is rendered locally by running JavaScript in the browser. This has a dramatic impact on site behavior, performance, and SEO.
+
+Rendering your HTML in the browser may sound like the faster option vs. requesting it from a remote server. However, a true SPAs (with no server rendering) will be consistently slower than an MPA. This is because an SPA has to download, parse, and run an entire JavaScript application in the browser just to render any HTML on the page. Then, your SPA will likely need to fetch remote data anyway, introducing even more wait time before your page is finished loading.
```

Comment:
```suggestion
Rendering your HTML in the browser may sound like the faster option vs. requesting it from a remote server. However, an true SPA (with no server rendering) will be consistently slower than an MPA. This is because an SPA has to download, parse, and run an entire JavaScript application in the browser just to render any HTML on the page. Then, your SPA will likely need to fetch remote data anyway, introducing even more wait time before your page is finished loading.
```